bodhi/server/models/models.py

```diff
@@ -1708,6 +1716,18 @@ def last_modified(self):
         return possibilities[-1]  # Return the last one
 
     @property
+    def has_negative_karma(self):
+        """Check for negative karma, Returns True if the update has negative karma"""
+        feedback = defaultdict(int)
+        for comment in self.comments:
+            if not comment.anonymous:
+                feedback[comment.user.name] += comment.karma
```

Comment:
It occurred to me that this code will not work correctly if the user gives positive karma and then changes their mind and posts negative karma. This function will count their karma as 0.